swagger.yaml

```diff
@@ -3741,6 +3741,10 @@ definitions:
         x-go-name: SNORequirements
         $ref: '#/definitions/cluster-host-requirements-details'
         description: Single node OpenShift node requirements
+      dpu-worker:
```

Comment:
not sure i like the dpu worker type. maybe call it edge-worker, and have a list of mappings?

Reply:
possible